cmd/crates/soroban-spec-typescript/fixtures/test_custom_types/README.md

```diff
@@ -6,7 +6,7 @@ This library was automatically generated by Soroban CLI using a command similar
 
 ```bash
 soroban contract bindings ts \
-  --rpc-url https://rpc-futurenet.stellar.org:443 \
+  --rpc-url https://rpc-futurenet.stellar.org:443/soroban/rpc \
```

Comment:
was this intentional ?

Reply:
hm, nope. not sure where that came from, actually. looking :detective:

Reply:
nice catch - looks like the boilerplate wasn't updated

Reply:
now I'm even more confused : you updated the boilerplate, but left the opposite in here ?!